.github/dependabot.yml

```diff
@@ -2,9 +2,9 @@
 version: 2
 
 updates:
-  # Check for updates to GitHub Actions.
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 10
+    # Check for updates to GitHub Actions.
+    - package-ecosystem: 'github-actions'
+      directory: '/'
+      schedule:
+          interval: 'daily'
+      open-pull-requests-limit: 10
```

Comment:
Are these changes necessary or are they an auto-format leftover?

Reply:
Oh, not sure where this came from! I'll reset the file.